libs/utils/Cargo.toml

```diff
@@ -1,17 +1,19 @@
 [package]
-name = "graph_utils"
+name = "utils"
```

Comment:
Hmm. I am guessing these utils are redundant at this point? Ah this is a little bit awkward, but I do not see a good alternative for the example filter. The graph utils should definitely be part of the compiler for the envoy filter..

Reply:
Yeah, they're redundant.  But if you don't copy them over, the tests unique to tracing_sim fail.  So it's a bit of an awkward way of going about it, but I can't see another way to test the simulator.